api/database/seeders/BigSeederPoolCandidateUser.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Enums\AssessmentResultType;
+use App\Enums\AssessmentStepType;
+use App\Enums\PoolSkillType;
+use App\Models\AssessmentResult;
+use App\Models\AssessmentStep;
+use App\Models\Community;
+use App\Models\Pool;
+use App\Models\PoolCandidate;
+use App\Models\PoolSkill;
+use App\Models\User;
+use Illuminate\Database\Seeder;
+
+class BigSeederPoolCandidateUser extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * This seeds a lot of data
+     * Run this AFTER core data has been seeded, this will not seed platform data
+     *
+     * User/PoolCandidate/Pool/AssessmentResult
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $input = $this->command->ask('Please enter how many times to loop');
+        $limit = intval($input);
+        $limit = (is_int($limit) && $limit > 0) ? $limit : 1;
+
+        // constant values for reuse and setup
+        $digitalCommunityId = Community::select('id')->where('key', 'digital')->sole()->id;
+        $atipCommunityId = Community::select('id')->where('key', 'atip')->sole()->id;
+        $financeCommunityId = Community::select('id')->where('key', 'finance')->sole()->id;
+
+        $pools = Pool::factory()
+            ->count(2)
+            ->withPoolSkills(4, 4)
+            ->withQuestions(2, 2)
+            ->published()
+            ->withAssessments(3)
+            ->create();
+        $poolIds = $pools->pluck('id')->toArray();
+
+        // User - non-government
+        for ($i = 0; $i < $limit; $i++) {
+            User::factory()
+                ->asApplicant()
+                ->withSkillsAndExperiences()
+                ->afterCreating(function (User $user) use ($poolIds) {
+                    $this->applyToPools($user, $poolIds);
+                })
+                ->count(100)
```

Comment:
I think this for loop kills the performance.
 we could eliminate this for loop and  provide the no. of users to be created directly like count(100 * limit )

Reply:
I was messing with a loop because of performance concerns, if you try and seed a bunch of users at once you run the risk of crashing. Or memory issues.
I think looping like chunking helps with that

Reply:
Might want your thoughts on performance, @petertgiles

Reply:
Bumped to 500 https://github.com/GCTC-NTGC/gc-digital-talent/pull/13478/commits/2576f2de944f4b46513e79067f38496acd63a8cf